pkg/aws/eni/node.go

```diff
@@ -431,7 +431,7 @@ func (n *Node) CreateInterface(ctx context.Context, allocation *ipam.AllocationA
 	if err != nil {
 		return 0,
 			unableToGetSecurityGroups,
-			fmt.Errorf("%s %s", errUnableToGetSecurityGroups, err)
+			fmt.Errorf("%s %w", errUnableToGetSecurityGroups, err)
```

Comment:
```suggestion
			fmt.Errorf("%s: %w", errUnableToGetSecurityGroups, err)
```

More in this file as well.

Reply:
Done.